### ISSUE TITLE
mIoU is 0.108 after training

### ISSUE THREAD

**Issue by licaizi** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#7]
so , I just get mIoU 0.108, it's really bad, how can I improve the performance? thank you

**Comment by zhengyang-wang** [comment]
I need more information to help you. The code is supposed to get mIoU around 75 for PASCAL VOC2012 val.

**Comment by licaizi** [comment]
您好，首先感谢您在百忙之中能够回复我的提问，您来自中国吧，看您的主页，以前是南京大学的，所以我就用中文了，是这样的，我下载完您的代码后，因为执行"python3 main.py"时有些图片找不到，所以我从别处找来了列表文件，然后执行了"python3 main.py"，在main中我设置了"encoder_name"为"res50"，“pretrain_file”为"./reference model/resnet_v1_50.ckpt", 当训练完20000步后，我进行了测试，执行"python3 main.py --option=test"， 得到的"Pixel Accuracy"是0.99，但是IoU是0.108，我仿照test()代码，添加了predict操作，试图预测一张图片，得到的结果很差，请问可能是什么原因呢，再次感谢您！

**Comment by warmestwind** [comment]
@CaiziLee 你的loss最后是多少呢？跟zhengyang的对比一下？
因为他用的是增强后的数据集，所以我们使用的时候要核对一下自己的图片是不是跟txt里写的相对应。

**Comment by licaizi** [comment]
我的loss，最后是1.4多，请问数据集应该怎么操作啊，我想看看全设置一样的话，能不能跑出好结果，谢谢了

**Comment by zhengyang-wang** [comment]
To get the same result, you need to make sure your dataset is the same. The augmented PASCAL VOC 2012 dataset with 10582 images for training and 1449 images for validation. Here is a download link of augmented labels: https://www.dropbox.com/s/oeu149j8qtbs1x0/SegmentationClassAug.zip?dl=0

**Comment by licaizi** [comment]
@zhengyang-wang 谢谢您提供的link，我用deeplab_resnet_init.ckpt这个预训练模型加载训练，得到的loss为1.3+，‘--option=test’的mIoU为0.68+，用deeplab_resnet.ckpt得到的loss也是1.3+，'--option=test'的mIoU为0.769，加载resnet_v1_101.ckpt，'--option=test'只得到了0.567，不知道我的结果是不是正常的

**Comment by zhengyang-wang** [comment]
The loss seems correct. But the mIoU is lower than what I got. Could you please provide your GPU information and the log.txt of setting "deeplab+deeplab_resnet_init.ckpt"?

To help detecting the problem, I provide my final model here. https://www.dropbox.com/s/61b0cvk8fktgbaw/model.zip?dl=0
You can extract the files and put them in the model folder, then run "python main.py --option=test". Let's see if the results are the same.

**Comment by licaizi** [comment]
@zhengyang-wang Thank you for your reply,  my GPU : 1080ti, I don't know what's going on, but I can get a good result now which mIoU is 0.753, I tried to train it for several times, and always 0.75+,  thank you for your help again

**Comment by Kewenjing1020** [comment]
I use resnet-50.ckpt for finetuning and only get mIoU as 0.674. How do you guys get more than 0.75? Do you use resnet-101? And what is your training steps? @CaiziLee @zhengyang-wang
